.circleci/config.yml

```diff
@@ -127,7 +127,14 @@ commands:
       - store_artifacts:
           path: tests/logs
 
-  platforms-build-steps-cpu:
+  platforms-build-matrix-steps:
+    parameters:
+      lite:
+        type: string
+      osnick:
+        type: string
+      target:
+        type: string
```

Comment:
add comments explaining what is what?

Reply:
also, where is platform-build-matrix-steps is used?

Reply:
Removed in push, but also documented variables in place with comments.